sdks/python/apache_beam/io/gcp/bigquery_schema_tools.py

```diff
@@ -76,11 +79,11 @@ def generate_user_type_from_bq_schema(the_table_schema):
 
 
 def bq_field_to_type(field, mode):
-  if mode == 'NULLABLE':
+  if mode == 'NULLABLE' or mode is None or mode == '':
```

Comment:
what's the motivation behind this change? I wonder if it could be a breaking change for some user pipelines.

Reply:
I wonder if it could be a breaking change for some user pipelines, because some hints downstream are more restrictive.

>>It won't be a breaking change since this solely applies to pipelines utilizing the `BEAM_ROW` output type (which none do yet). The motivation is to make the default mode Nullable ([as done in the Java sdk](https://developers.google.com/resources/api-libraries/documentation/bigquery/v2/java/latest/com/google/api/services/bigquery/model/TableFieldSchema.html#setMode-java.lang.String-)).